django/contrib/sessions/backends/base.py

```diff
@@ -161,10 +161,25 @@ def _get_or_create_session_key(self):
             self._session_key = self._get_new_session_key()
         return self._session_key
 
+    def _validate_session_key(self, key):
+        """Key must be truthy and at least 8 characters long"""
```

Comment:
Added periods to docstrings.
The 8 character limitation is indeed an arbitrary lower bound to enforce a minimum level of collision security. I went with 8 as it was suggested in the trac ticket.